lib/checking_account.rb

```diff
@@ -0,0 +1,75 @@
+require_relative './account'
+require 'csv'
+
+module Bank
+  class MoneyMarketAccount < Bank::Account
+    def initialize(id, balance)
+      super(id, balance)
+      @counter = 0
+    end
+
+    def withdraw(amount)
```

Comment:
It might be cleaner to use `super` here to call `Account#withdraw`, and centralize your logic all in one place. In this case `Account#withdraw` would probably require more parameters, like a fee and a minimum balance.